docs/ci-performance.md

```
@@ -36,7 +36,24 @@
 4. Set `run_performance` to `true`.
 5. Set `force_baseline` to `true` if you want to re-run baseline benchmarks.
 
-## Notes
+## Regression Detection Thresholds
 
-- This mechanism ensures efficient CI runs and gives maintainers control over when to refresh baseline results for performance validation.
-- For more details on running benchmarks locally, see [build/scripts/perf/README.md](../build/scripts/perf/README.md).
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)